src/domain/community/community/CommunityUpdatesDialog/CommunityUpdatesDialog.tsx

```diff
@@ -1,31 +1,42 @@
 import React, { FC } from 'react';
 import { useTranslation } from 'react-i18next';
 import Dialog from '@mui/material/Dialog';
-import { Box, Button, DialogActions } from '@mui/material';
-import { DialogContent, DialogTitle } from '../../../../core/ui/dialog/deprecated';
+import { Box, Button, DialogActions, IconButton } from '@mui/material';
+import { DialogContent } from '../../../../core/ui/dialog/deprecated';
 import { CommunityUpdatesContainer } from '../../../communication/updates/CommunityUpdatesContainer/CommunityUpdatesContainer';
 import { CommunityUpdatesView } from '../views/CommunityUpdates/CommunityUpdatesView';
+import DialogHeader from '../../../../core/ui/dialog/DialogHeader';
+import ShareButton from '../../../shared/components/ShareDialog/ShareButton';
+import { Close } from '@mui/icons-material';
 
 export interface CommunityUpdatesDialogProps {
   open: boolean;
   onClose: () => void;
   spaceId?: string;
   communityId?: string;
+  shareUrl: string;
 }
 
-const CommunityUpdatesDialog: FC<CommunityUpdatesDialogProps> = ({ open, onClose, spaceId, communityId = '' }) => {
+const CommunityUpdatesDialog: FC<CommunityUpdatesDialogProps> = ({
+  open,
+  onClose,
+  spaceId,
+  communityId = '',
+  shareUrl,
+}) => {
   const { t } = useTranslation();
-  const handleClose = () => {
-    onClose();
-  };
 
   return (
     <Dialog open={open} maxWidth="md" fullWidth aria-labelledby="community-updates-dialog-title">
-      <DialogTitle id="community-updates-dialog-title" onClose={handleClose}>
-        <Box display="flex" alignItems="center">
-          {t('dashboard-updates-section.dialog-title')}
-        </Box>
-      </DialogTitle>
+      <DialogHeader
+        actions={
+          <>
+            <IconButton onClick={onClose}>{<Close />}</IconButton>
```

Comment:
probably you have copied this from somewhere else, but I think it's better to use the `onClose` property of the DialogHeader, the result will be the same but you can remove `IconButton` and `CloseIcon` imports from here if you do it